src/blocks/SliderNew/schema.ts

```diff
@@ -0,0 +1,93 @@
+import {
+    AnimatableProps,
+    BlockBaseProps,
+    BlockHeaderProps,
+    ChildrenCardsProps,
+    sliderSizesObject,
+    textSize,
+} from '../../schema/validators/common';
+
+const LoadableProps = {
+    additionalProperties: false,
+    required: ['source'],
+    properties: {
+        source: {
+            type: 'string',
+            // add loadable sources here if you use it in your project
+            // enum: ['my-loadable-source-1', 'my-loadable-source-1'],
```

Comment:
Is this really necessary here?

Reply:
Let's keep these comments (this one and the one below) as long as slider-new retains this logic from old slider